distribution/ddf-common/src/main/resources/bin/setenv.bat

```diff
@@ -58,7 +58,7 @@ rem SET KARAF_BASE
 rem Additional available Karaf options
 rem SET KARAF_OPTS=
 
-rem comment out the line below to enable cxf logging interceptors
+rem Uncomment out the line below to enable cxf logging interceptors
```

Comment:
❓ Windows?

Reply:
Simple update to a comment that was wrong to start with.